Source/WebCore/page/csp/ContentSecurityPolicy.cpp

```diff
@@ -1189,4 +1189,16 @@ void ContentSecurityPolicy::setInsecureNavigationRequestsToUpgrade(HashSet<Secur
     m_insecureNavigationRequestsToUpgrade = WTFMove(insecureNavigationRequests);
 }
 
+const HashAlgorithmSetCollection& ContentSecurityPolicy::hashesToReport()
+{
+    if (m_hashesToReport.isEmpty()) {
+        Vector<std::pair<HashAlgorithmSet, FixedVector<String>>> hashesToReport;
+        for (auto& policy : m_policies) {
+            if (auto hash = policy->reportHash())
+                hashesToReport.append(std::make_pair(hash, policy->reportToTokens()));
+        }
+        m_hashesToReport = hashesToReport;
```

Comment:
This should be `m_hashesToReport =  WTFMove(hashesToReport)` for better efficiency, so we move the fixed vectors of strings instead of copying them.

Reply:
of course. done!